src/qtbase/QSizeF.js

```diff
@@ -0,0 +1,7 @@
+class QSizeF extends QObject {
```

Comment:
This is my first time using "real" JavaScript class inheritance, so please let me know if there are any issues.